user_manual/files/access_webgui.rst

```diff
@@ -193,41 +193,65 @@ with the user "pierpont", who can *share*, *edit*, *create*, *change*, and
 .. figure:: ../images/files_page-2.png
    :alt: Sharing files.
 
+Sharing Files with Guest Users
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Users can also share files and folders with guest users. 
+To do so, your ownCloud administrator will need to have installed the Guest application.
+
+If it's already installed, in the "**User and Groups**" field of the Sharing panel, type the username or email address of a user who is not already a user in your ownCloud installation.
```

Comment:
Ok, I'll make an adjustment. That distinction wasn't clear when I was working with the UI.